internal/goast/package.go

```diff
@@ -35,7 +36,15 @@ func DeterminePackageName(importPath string) string {
 
 	pkg, err := build.Import(importPath, "", 0)
 	if err != nil {
-		return filepath.Base(importPath)
+		return guessPackageName(importPath)
 	}
 	return pkg.Name
 }
+
+func guessPackageName(importPath string) string {
+	packageName := filepath.Base(importPath)
+	if strings.HasSuffix(packageName, "-go") {
```

Comment:
It's not necessary at all. It's just a nice-to-have for the common pattern of "foo-go" packages using the name "foo". We'll still use a qualified import for this because "foo-go" is not equal to "foo".